UKHomeOffice/nmsw-ui
# NMSW-324 Add supporting doc route

# Ticket

NMSW-324

----

## AC



Page | URL | Link type | Link name | Link route | Next page
-- | -- | -- | -- | -- | --
Upload supporting docs | /report-voyage/upload-supporting-documents | tbd | tbd | tbd | tbd



----

## To test

- go to /report-voyage/upload-supporting-documents
- > a h1 should show on the page

----

## Notes
We don't have final requirements for page content or layout so there is just a bare page with a h1 and no tests for it yet.
